docs/content/en/release-notes/_index.md

```diff
@@ -39,18 +39,19 @@ The K8ssandra helm chart deploys the following components. Some are optional, an
   * 3.11.8
   * 3.11.9
   * 3.11.10 (default)
+  * 4.0 rc1
 * DataStax Kubernetes Operator for Apache Cassandra ([cass-operator](https://github.com/k8ssandra/cass-operator)) 1.7.0
 * Management API for Apache Cassandra ([MAAC](https://github.com/datastax/management-api-for-apache-cassandra)) 0.1.24
 * [Stargate](https://github.com/stargate/stargate) 1.0.18
 * Metric Collector for Apache Cassandra ([MCAC](https://github.com/datastax/metric-collector-for-apache-cassandra)) 0.2.0
 * kube-prometheus-stack 12.11.3 [chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
-* Medusa for Apache Cassandra 0.10.0
+* Medusa for Apache Cassandra 0.10.1
 * Reaper for Apache Cassandra 2.2.2
 
 ## Upgrade notice
 
 {{% alert title="Important!" color="warning" %}}
-Upgrading from K8ssandra 1.0.0 to 1.1.0 causes a StatefulSet update (due to [#533](https://github.com/k8ssandra/k8ssandra/issues/533) and [#613](https://github.com/k8ssandra/k8ssandra/issues/613)). A StatefulSet update has the effect of a rolling restart. Because of [#411](https://github.com/k8ssandra/k8ssandra/issues/411) this could require you to perform a manual restart of all Stargate nodes after the Cassandra cluster is back online. This behavior also impacts in-place restore operations of Medusa backups [#611](https://github.com/k8ssandra/k8ssandra/issues/611). 
+Upgrading from K8ssandra 1.0.0 to 1.1.0, or directly from K8ssandra 1.0.0 to 1.2.0 (skipping 1.1.0), causes a StatefulSet update (due to [#533](https://github.com/k8ssandra/k8ssandra/issues/533) and [#613](https://github.com/k8ssandra/k8ssandra/issues/613)). A StatefulSet update has the effect of a rolling restart. Because of [#411](https://github.com/k8ssandra/k8ssandra/issues/411) this could require you to perform a manual restart of all Stargate nodes after the Cassandra cluster is back online. This behavior also impacts in-place restore operations of Medusa backups [#611](https://github.com/k8ssandra/k8ssandra/issues/611). 
```

Comment:
Do we have guidance on how to restart Stargate nodes?

Reply:
Chris, do you mean guidance beyond the 4 steps below that paragraph?  It's not shown in these DIFFs because we listed them in the 1.1.0 RNs.  https://docs.k8ssandra.io/release-notes/#upgrade-notice